test/scripts/test_isolate_tests.py

```diff
@@ -6,57 +6,111 @@
 
 # NOTE: This test file file only works with scripts/ added to PYTHONPATH so pylint can't find the imports
 # pragma pylint: disable=import-error
-from isolate_tests import extract_docs_cases
+from isolate_tests import extract_solidity_docs_cases, extract_yul_docs_cases
 # pragma pylint: enable=import-error
 
+import textwrap
 
 CODE_BLOCK_RST_PATH = FIXTURE_DIR / 'code_block.rst'
 CODE_BLOCK_RST_CONTENT = load_fixture(CODE_BLOCK_RST_PATH)
 CODE_BLOCK_WITH_DIRECTIVES_RST_PATH = FIXTURE_DIR / 'code_block_with_directives.rst'
 CODE_BLOCK_WITH_DIRECTIVES_RST_CONTENT = load_fixture(CODE_BLOCK_WITH_DIRECTIVES_RST_PATH)
 
+# Formats code to contain only one indentation and terminate with a \n
+def formatCase(text):
+    return textwrap.indent(textwrap.dedent(text.lstrip("\n")), "    ") + "\n"
```

Comment:
That's a nice improvement. I originally did not want to bother because of the indent but it turned out simpler than I would expect :)

Reply:
By the way, to make the comment a docstring, put it inside the function (see [PTP-257](https://www.python.org/dev/peps/pep-0257/)):
```suggestion
def formatCase(text):
    """Formats code to contain only one indentation and terminate with a \n."""
    return textwrap.indent(textwrap.dedent(text.lstrip("\n")), "    ") + "\n"
```